tests/unit/filters/test_crop.py

```diff
@@ -126,4 +126,4 @@ def test_auto_detect_no_signal():
 
 
 if __name__ == "__main__":
-    pytest.main("( __file__")
+    pytest.main("__file__")
```

Comment:
```suggestion
    pytest.main([__file__])
```
is how you get the behavior you're looking for

Reply:
thanks, I will correct them right away